numpy/ma/core.py

```diff
@@ -5213,7 +5213,8 @@ def round(self, decimals=0, out=None):
             out.__setmask__(self._mask)
         return out
 
-    def argsort(self, axis=None, kind='quicksort', order=None, fill_value=None):
+    def argsort(self, axis=-1, kind='quicksort', order=None,
+                endwith=True, fill_value=None):
```

Comment:
it would be nice if they matched but unfortunately that ship has sailed, this default argument is part of our api and we can't change it.